.github/workflows/publish.yml

```
@@ -0,0 +1,34 @@
+name: Publish
+
+on:
+  release:
+    types: [published, edited]
```

I thought we had the convention to always use the dash format for yaml arrays:

```suggestion
    types:
      - published
      - edited
```